content/partials/app-preview-builds.md

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+Codemagic allows you to preview your unsigned `.app` artifact on an iOS simulator and interact with the simulator via a web browser. You can easily test the functionalities of your app as well as system notifications and flows that require location data on different simulator devices without requiring a physical device.
```

Comment:
Apps can be signed too, it is just unlikely that anyone would go through the trouble.

```suggestion
Codemagic allows you to preview your `.app` artifact on an iOS simulator and interact with the simulator via a web browser. You can easily test the functionalities of your app as well as system notifications and flows that require location data on different simulator devices without requiring a physical device.
```